nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/hesai_decoder.hpp

```diff
@@ -36,6 +37,52 @@
 namespace nebula::drivers
 {
 
+struct HesaiDecodeFilteredInfo
```

Comment:
Please also add counters for 
* invalid points (distance == 0), line 198
* points filtered due to dual_return_threshold (line 238)
* points filtered due to identical return type (line 217)
* points kept in total (= not filtered) (shall be equal to the final pointcloud size)